2021/1001xxx/GSD-2021-1001147.json

```diff
@@ -9,21 +11,30 @@
     "impact": "SQL Injection",
     "credit": "",
     "references": [
-      "https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0017"
+      {
+        "type": "WEB",
+        "url": "https://psirt.global.sonicwall.com/vuln-detail/SNWLID-2021-0017"
+      }
+    ],
+    "affected": [
+      {
+        "package": {
+          "name": "Secure Remote Access (SRA)",
+          "ecosystem": "GSD"
```

Comment:
Wait, isn't the ecosystem supposed to be the ecosystem of the package? e.g., NPM or RubyGems.

Reply:
There isn't an ecosystem for general software so we use "GSD" https://ossf.github.io/osv-schema/#affectedpackage-field

Reply:
This is an example of the limitations of ecosystem / package, relevant to:

https://github.com/ossf/osv-schema/issues/83

Reply:
That's not compliant with the OSV schema

Reply:
Neither is using an ecosystem of GSD.

Reply:
Oliver had a suggestion for what to temporarily use for unsupported ecosystems somewhere, let me see if I can find it again.

Reply:
https://github.com/github/advisory-database/discussions/568#discussioncomment-3330460

Reply:
Although also not officially OSV.  Has someone already started a conversation on the OSV spec either proposing new ecosystem values or having some way of allowing unsupported ones until there is an official one?

Reply:
Yes: https://github.com/ossf/osv-schema/issues/83